dhall-openapi/openapi-to-dhall/Main.hs

```diff
@@ -31,7 +31,7 @@ import Dhall.Kubernetes.Types
     , Prefix
     , Swagger (..)
     )
-import System.FilePath                  (FilePath, (</>))
+import System.FilePath                  ((</>))
```

Comment:
I was surprised that this one wasn't caught via `failOnAllWarnings` in the Nix build.

Reply:
Yeah, I'm not sure either.  Maybe the redundant import checker's behavior changed between GHC versions